test/unit/tasks/cms/cms_test.rb

```diff
@@ -2,24 +2,56 @@
 
 module Tasks
   class CmsTest < ActiveSupport::TestCase
+    class CmsTestFixPaginationHref < CmsTest
+      def setup
+        @page = FactoryBot.build(:cms_partial, system_name: 'shared/pagination',
+                                 published: '<p>Title</p><a href="{{ part.link }}">Supertramp</a>')
```

Comment:
This indentation hurts my eyes 😄. However, I just learnt it is the standard for ruby actually.

Reply:
Your eyes are wrong